engine/runtime-language-arrow/src/main/java/org/enso/interpreter/arrow/util/MemoryUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.enso.interpreter.arrow.util;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.nio.ByteBuffer;
+
+public class MemoryUtil {
+
+  private static Constructor<?> byteBufferConstr = null;
+
+  static {
+    ByteBuffer buffer = null;
+    try {
+      buffer = ByteBuffer.allocateDirect(1);
+      byteBufferConstr = buffer.getClass().getDeclaredConstructor(long.class, long.class);
+      byteBufferConstr.setAccessible(true);
+    } catch (NoSuchMethodException e) {
+      System.err.println(
```

Comment:
Maybe throw `IllegalStateException` instead of just printing to stderr? Not being able to find a constructor of `DirectoByteBuffer` is a fatal error, no?